snippets/quick-order-list-row.liquid

```diff
@@ -13,6 +13,12 @@
 {% assign cart_qty = cart | item_count_for_variant: variant.id %}
 {% # theme-check-enable %}
 
+{%- capture escaped_options -%}
+  {% for option in variant.options -%}
+    {{ option | escape }}{%- unless forloop.last -%}{{ ' / ' }}{%- endunless -%}
+  {%- endfor %}
+{%- endcapture -%}
+
```

Comment:
This change has no visual impact for normal products, but for a combined listing the result is that both the parent and child option values will be displayed in the quick order list.

For example, for a Pants CL product with a child `Pants - black` and option `size`, this will display `Black / Large` instead of just `Large`.